content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -252,16 +252,7 @@ The following constraints apply when you combine `groupBy` and `orderBy`:
 
 #### Order by aggregate group
 
-In [2.21.0](https://github.com/prisma/prisma/releases/2.21.0) and later, you can **order by aggregate group**.
```

Comment:
This should probably not be removed, as for users on older versions before 2.21.0 it is relevant to know that this will not work for them. Possibly we could even change this to include that with 3.4.0 this also works for MongoDB.

Reply:
Thanks @Jan, I will add back in the version #s, then going forward let's think on how we want to handle this.